src/api.js

```diff
@@ -823,6 +823,11 @@ API.prototype.router = function(options) {
       'Context must have declared property: \'' + property + '\'');
   });
 
+  Object.keys(options.context).forEach(function(property) {
+    assert(this._options.context.indexOf(property) !== -1,
+      'Context has unexpected property: \'' + property + '\'');
```

Comment:
You can use a string template for this: `Context has unexpected property '${property}'`.